experimenter/experimenter/experiments/changelog_utils.py

```diff
@@ -87,3 +92,66 @@ def generate_nimbus_changelog(experiment, changed_by, message, changed_on=None):
         message=message,
         changed_on=changed_on,
     )
+
+
+def get_formatted_change_object(field_name, field_diff, changelog, timestamp):
+    event_name = ChangeEventType.GENERAL.name
+    field_instance = NimbusExperiment._meta.get_field(field_name)
+    field_display_name = (
+        field_instance.verbose_name
+        if hasattr(field_instance, "verbose_name")
+        else field_name
+    )
```

Comment:
This is a perfect case for [`getattr`](https://docs.python.org/3/library/functions.html#getattr)

```suggestion
    field_display_name = getattr(field_instance, "verbose_name", field_name)
```